package.json

```diff
@@ -54,8 +64,24 @@
     ]
   },
   "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
+    "hooks": {}
+  },
+  "jest": {
+    "coveragePathIgnorePatterns": [
```

Comment:
I think you should remove this `coveragePathIgnorePatterns` config, because is used for ignore files that are not necessary related with the app, but on this case, the next files should not be here:

```
"/src/providers",
"/src/providers",
"/src/config",
"/src/helpers/",
"/src/utils/",
"/src/index.js",
"/src/components/Fortune",
 "/src/components/Private",
"/pages/NotFound",
"/components/App/"
```

Please remove and update the code coverage report.